modules/rules_swift/1.2.0/presubmit.yml

```diff
@@ -0,0 +1,28 @@
+shell_commands: &shell_commands
+- "echo --- Downloading and extracting Swift $SWIFT_VERSION to $SWIFT_HOME"
+- "mkdir $SWIFT_HOME"
+- "curl https://download.swift.org/swift-${SWIFT_VERSION}-release/ubuntu2004/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu20.04.tar.gz | tar xvz --strip-components=1 -C $SWIFT_HOME"
+
+matrix:
+  platform:
+  - ubuntu2004
```

Comment:
Once https://github.com/bazelbuild/rules_swift/pull/832 is merged, we could try adding Windows here as well.